src/System.Data.SqlClient/src/System/Data/SqlClient/SqlException.cs

```diff
@@ -11,25 +11,47 @@
 using System.Data.Common;
 using System.Diagnostics;
 using System.Globalization;
+using System.Runtime.Serialization;
 using System.Text; // StringBuilder
 
 namespace System.Data.SqlClient
 {
+    [Serializable]
     public sealed class SqlException : System.Data.Common.DbException
```

Comment:
>SqlException [](start = 24, length = 12)

should this be in your SerializableExceptions() test in s.runtime.serialization.formatters?